### PR TITLE
DEVDOCS-2051: Dev Center | Theme Docs | Installing Stencil CLI

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2051](https://jira.bigcommerce.com/browse/DEVDOCS-2051)

## What changed?
* Sentence Case
* Grammar

### CODE REVIEW

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
 - [Installing on Windows](#installing-on-windows)
 - [Installing on Linux](#installing-on-linux)
-- [Live Previewing a Theme](#live-previewing-a-theme)
+- [Live previewing a theme](#live-previewing-a-theme)
 - [Resources](#resources)
 
 </div>
 
-Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and its built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands. 
+Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and it's built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands.
```
Comment (traci707) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:14]: "Its" is referring to Stencil CLI,  not it is (or it's) built-in 
```suggestion
Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and it's built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands.

```

Comment (traci707) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:14]:
```suggestion
Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and it's built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands.

```
"Its" is referring to Stencil CLI. It should not be it's (or it is) built-in Browswersync...

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
 
 This article contains the detailed instructions needed to install and configure Stencil CLI -- the first step to developing themes on the BigCommerce platform.
 
 ## Installing on Mac
 
-To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands: 
+To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands:
```
Comment (traci707) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:20]:
```suggestion
To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands:

```
"To install Stencil CLI and its dependencies..."

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 3]
```diff
@@ -111,7 +111,7 @@ npm install -g @bigcommerce/stencil-cli
 
 <!-- theme: info -->
 
-> These instructions have been tested successfully on **Windows 10**. 
+> These instructions have been tested successfully on **Windows 10**.
```
Comment (traci707) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:114]:
```suggestion
> These instructions have been tested successfully on **Windows 10**.

```
Passive voice

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 4]
```diff
@@ -148,13 +148,13 @@ npm install -g @bigcommerce/stencil-cli
 
 <!-- theme: info -->
 
-> These instructions have been tested on **Ubuntu 18.04**. 
+> These instructions have been tested on **Ubuntu 18.04**.
```
Comment (traci707) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:151]:
```suggestion
> These instructions have been tested on **Ubuntu 18.04**.

```
Passive voice

**Comment by aglensmith** [comment]
changes applied

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 5]
```diff
@@ -2,22 +2,24 @@
...
+Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and it's built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands.
+
 
 This article contains the detailed instructions needed to install and configure Stencil CLI -- the first step to developing themes on the BigCommerce platform.
 
 ## Installing on Mac
 
-To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands: 
+To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands:
```
Comment (uwikaiddi) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:21]:
```suggestion
To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands.

```

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 6]
```diff
@@ -80,17 +82,17 @@ npm install -g @bigcommerce/stencil-cli
...
+>
+### Chocolatey installation alternatives
 > For additional information on installing Chocolatey and alternative installation options, see [the installation page on chocolatey.org](https://chocolatey.org/install).
 
 </div>
 </div>
 </div>
 
-### Method 2: Install Dependencies Manually
+### Method 2: install dependencies manually
```
Comment (uwikaiddi) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:95]:
```suggestion
### Method 2: Install dependencies manually

```

Comment (uwikaiddi) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:95]:
```suggestion
### Method 2: Install dependencies manually

```

**Pull request merged**